.circleci/config.yml

```diff
@@ -92,9 +92,6 @@ commands:
           name: Follow logs, to help diagnose failures
           command: make logs
           background: true
-      - run:
```

Comment:
this takes 1m50s and all these tests are also run by `test-e2e'. Hopefully we can reduce e2e time to 8m